.github/workflows/main.yml

```diff
@@ -19,6 +19,9 @@ jobs:
         with:
           nix_path: nixpkgs=channel:nixos-unstable
 
+      - name: Check flake
```

Comment:
Shouldn't this be tested at the very end? At least after testing. So we avoid to wait 2 minutes to get result on the actual changes.

Reply:
Do we still need to do bob build if nix build fails?

Reply:
Actually it makes sense like you say because we are more interested on our changes. Will move